### PR TITLE
[MCC-202660] Update documentation and NuGet

### PR DESCRIPTION
**Pull request by lschreck-mdsol**
@kenyamat @bvillanueva-mdsol Sorry I forgot to update the documentation in my previous PR (#72).

This PR includes the Readme.md file and also updates the NuGet version to the latest (and make the package generator batches work with the latest MSBuild).

As these changes are affecting only the source (documentation) and the build, there is no need to generate new NuGet packages.

Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,14 +153,21 @@ using (var client = new HttpClient())
...
     tracerClient.EndClientTrace(span);	
 }
 ...
 ```
 
+In case of the `ZipkinClient.Record()` method, the second parameter(`value`) can be omitted during the call, in that case the caller member name (method, property etc.) will get recorded.
+
 ## Contributors
 ZipkinTracer is (c) Medidata Solutions Worldwide and owned by its major contributors:
 * Tomoko Kwan
```
Comment (bvillanueva-mdsol) [inline README.md]: @lschreck-mdsol please add your name here. Thanks

File: README.md [hunk 2]
```diff
@@ -114,8 +115,32 @@ context.EndRequest += (sender, args) =>
...
+    result = await client.GetAsync(requestUri);
+
+	// end client trace
+    tracerClient.EndClientTrace(span);	
+}
+...
+```
+
+#### Recording arbitrary events and additional information
+Additional annotations can be recorded by using the ZipkinClient's `Record()` and `RecordBinary&lt;T&gt;()` methods:
```
Comment (bvillanueva-mdsol) [inline README.md]: This does not display correctly in preview --> "RecordBinary<T>()"

Comment (lschreck-mdsol) [inline README.md]: Thanks, I was not sure how to escape the <> signs, but it seems that Github will ignore it as a HTML and render it correctly.

File: README.md (hunk 1 above)
Comment (lschreck-mdsol) [inline README.md]: Done! No more excuses for me to contribute here :stuck_out_tongue_closed_eyes:

**Comment by bvillanueva-mdsol** [comment]
LGTM.

**Comment by bvillanueva-mdsol** [comment]
merging.. thanks.

**Comment by bvillanueva-mdsol** [comment]
As per my understanding, this does not include the lc solution yet, this is just for the custom annotations. @lschreck-mdsol will follow up on another pr for the local trace solution 
@jcarres-mdsol @kenyamat @cabbott

**Comment by lschreck-mdsol** [comment]
@bvillanueva-mdsol @jcarres-mdsol please refer to the #74 

Thank you!

**Pull request merged**